.github/workflows/tests.yml

```diff
@@ -80,6 +76,10 @@ jobs:
           key: ${{ runner.os }}-composer-${{ matrix.php-version }}-${{ matrix.symfony-version }}-${{ hashFiles('composer.json') }}
           restore-keys: ${{ runner.os }}-composer-${{ matrix.php-version }}-${{ matrix.symfony-version }}-
 
+      - name: Allow Composer Plugins
+        run: |
+          composer config --no-plugins allow-plugins.ocramius/package-versions true
```

Comment:
What is the intent of this change?

Reply:
Otherwise i get the following error. Maybe this is only for runs on pull requests?
https://github.com/liip/LiipTestFixturesBundle/actions/runs/6841422883/job/18601768024

```
Symfony on PHP 7.4 flags update --prefer-lowest
ocramius/package-versions contains a Composer plugin which is blocked by your allow-plugins config. You may add it to the list if you consider it safe.
You can run "composer config --no-plugins allow-plugins.ocramius/package-versions [true|false]" to enable it (true) or disable it explicitly and suppress this exception (false)
See https://getcomposer.org/allow-plugins

```

Reply:
I have readded this paragraph to make CI green.

Reply:
I need to bump more dev dependencies to make ci green for `--prefer-lowest`

Reply:
You can run `composer config --no-plugins allow-plugins.ocramius/package-versions true` on your branch then commit the `composer.json` file. So that contributors of this bundle won't have the same error message. Then remove the change in `.github/workflows/tests.yml`.

Reply:
Yes i can do this.
Some maintainers, want to have this in composer.json, and others only in the ci.
https://github.com/ohdearapp/health-check-results/pull/5

Reply:
Done